whylog/config/investigation_plan.py

```diff
@@ -102,3 +102,10 @@ def __init__(self, host, path):
 
     def __repr__(self):
         return "(LineSource: %s:%s)" % (self.host, self.path)
+
+    def __eq__(self, other):
+        if self is None and other is None:
+            return True
+        if self is None or other is None:
+            return False
+        return all([self.host == other.host, self.path == other.path])
```

Comment:
don't allocate a list here. Use generator expression instead of list comprehension.